diff --git a/bin/emHdBindings.worker.js b/bin/emHdBindings.worker.js
--- a/bin/emHdBindings.worker.js
+++ b/bin/emHdBindings.worker.js
@@ -68,25 +68,6 @@
           postMessage({ cmd: "callHandler", handler: handler, args: args });
         };
       }
-      Module["urlCallbackFromWorker"] = async (...args) => {
-        postMessage({
-          cmd: "callHandlerAsync",
-          handler: "urlModifier",
-          args: args,
-        });
-        const promise = new Promise((resolve) => {
-          self.onmessage = (e) => {
-            if (
-              e.data.cmd === "callHandlerAsyncResult" &&
-              e.data.handler === "urlModifier"
-            ) {
-              self.onmessage = handleMessage;
-              resolve(e.data.result);
-            }
-          };
-        });
-        return await promise;
-      };
       Module["wasmMemory"] = e.data.wasmMemory;
       Module["buffer"] = Module["wasmMemory"].buffer;
       Module["ENVIRONMENT_IS_PTHREAD"] = true;
